Makefile

```diff
@@ -0,0 +1,66 @@
+TS = $(shell date +%Y%m%d%H%M%S)
```

Comment:
we can benefit from making the TS more readable instead of a string of numbers.
for example, separate date and time with a "_"
```
$ date +%Y%m%d_%H%M%S
20220110_141848
```

Reply:
Just realized that `TS` is used as a default value for `REVISION`, and `REVISION` can be passed as an env var to the makefile, so we can make this change in the Jenkins job directly. Please ignore this suggestion. :)